docs/source/07_extend_kedro/03_custom_datasets.md

```diff
@@ -168,16 +175,11 @@ Similarly, we can implement the `_save` method as follows:
 
 
 ```python
-import numpy as np
-from PIL import Image
-from kedro.io.core import AbstractDataSet, get_filepath_str
-
```

Comment:
Did you remove these because they are already included in the code snippet above?

Reply:
Yes, exactly. We weren't being consistent about whether we repeated the imports in every example or not. I simplified it by just giving them in the first example and the complete example.